app/components/authorShowHeader/index.tsx

```diff
@@ -124,16 +124,19 @@ class AuthorShowHeader extends React.PureComponent<AuthorShowHeaderProps, Author
               </a>
             )}
         </div>
-        {author.email && (
+        {author.isEmailHidden ? (
```

Comment:
`author.email` can be `null` too. 
`!author.isEmailHidden && author.email &&` will be great.